rosbag2_storage_mcap/include/rosbag2_storage_mcap/message_definition_cache.hpp

```diff
@@ -84,15 +89,15 @@ class MessageDefinitionCache final {
    * Load and parse the message file referenced by the given datatype, or return it from
    * msg_specs_by_datatype
    */
-  const MessageSpec& load_message_spec(const DefinitionIdentifier& definition_identifier);
+  const MessageSpec & load_message_spec(const DefinitionIdentifier & definition_identifier);
 
   std::unordered_map<DefinitionIdentifier, MessageSpec, DefinitionIdentifierHash>
     msg_specs_by_definition_identifier_;
 };
 
 ROSBAG2_STORAGE_MCAP_PUBLIC
-std::set<std::string> parse_dependencies(Format format, const std::string& text,
-                                         const std::string& package_context);
+std::set<std::string> parse_dependencies(
+  Format format, const std::string & text, const std::string & package_context);
```

Comment:
@james-rms I am curious if it's possible to preserve original formatting with indentation on second line with argument?

Reply:
We can if we remove the `AlignAfterOpenBracket: AlwaysBreak` line from `.clang-format`. However I'm not sure if i'm breaking some rule by doing this that's an explicit directive in `ament_clang_format`'s configuration, so i feel like it's there for a reason.